tests/test_other.py

```diff
@@ -7123,9 +7123,19 @@ def build(args=[]):
     # adding --metrics should not affect code size
     self.assertEqual(base_size, os.path.getsize('a.out.wasm'))
 
+  def check_expected_size_in_file(self, filename, size):
+    if common.EMTEST_REBASELINE:
+      with open(filename, 'w') as f:
+        f.write(f'{size}\n')
+    size_slack = 0.05
+    expected_size = int(read_file(filename).strip())
+    delta = size - expected_size
+    ratio = abs(delta) / float(expected_size)
+    print('  seen wasm size: %d (expected: %d) (delta: %d), ratio to expected: %f' % (size, expected_size, delta, ratio))
```

Comment:
```suggestion
    print('  seen %s size: %d (expected: %d) (delta: %d), ratio to expected: %f' % (what, size, expected_size, delta, ratio))
```
and need to send "wasm" or "js" to this func